db/migrate/20171208081924_add_priority_to_change_content.rb

```diff
@@ -0,0 +1,5 @@
+class AddPriorityToChangeContent < ActiveRecord::Migration[5.1]
+  def change
+    add_column :content_changes, :priority, :integer, default: 0
+  end
+end
```

Comment:
Maybe we should add an index, too? It seems quite likely we'd want to query by `high` priority content changes only.

Reply:
I agree we're likely to need an index here, but I wanted to avoid making any "bigger" decisions regarding priorities as we haven't fully established how they're going to work yet.

Reply:
Ok, seems reasonable. I think there was talk of doing a larger review of which fields need indices, so we can leave this for now.